sample/src/main/java/io/github/boguszpawlowski/composecalendar/sample/CustomSelectionSample.kt

```diff
@@ -47,9 +47,15 @@ private class MonthSelectionState(
 @Composable
 private fun rememberMonthSelectionState(
   initialMonth: YearMonth = YearMonth.now(),
+  minMonth: YearMonth = initialMonth.minusMonths(10000),
+  maxMonth: YearMonth = initialMonth.plusMonths(10000),
```

Comment:
Could reuse DefaultCalendarMonthRange constant.

Reply:
@boguszpawlowski I don't mind reusing it, but that would mean to make it a public const. I didn't see any public const in the current library, so I chose to make it internal like the rest of the code. 

Do you want to make it public? I'll await your response before changing any code.